licenses/autogenerated/EUPL-1.1.json

```diff
@@ -9,6 +9,7 @@
         ],
         "name": "European Union Public License, Version 1.1",
         "keywords": [
+            "discouraged",
```

Comment:
Comparing master with the “Superseded licenses” category, I think the pattern is to set both `discouraged` and `obsolete` for these (like [here][2]).

[1]: https://opensource.org/licenses/category
[2]: https://github.com/OpenSourceOrg/licenses/blob/f7ff223f9694ca0d5114fc82e43c74b5c5087891/licenses/manual/apache.json#L29-L30

Reply:
Updated.